spec/models/requests/pickup_lookup_spec.rb

```diff
@@ -25,7 +25,10 @@
           "barcode" => "32101089306938",
           "enum" => "Bd.1, T.1",
           "copy_number" => "1",
-          "status" => "Charged"
+          "status" => "Charged",
+          "item_type" => "Gen",
+          "pickup_location_id" => "299",
```

Comment:
I would remove `"pickup_location_id" => "299",` to test that the default works correctly.

Reply:
I tested it by setting it to nil and it returned 299. Isn't this what we want to fall back to 299 if there is no value?

Reply:
@carolyncole I updated the spec by removing the 'pickup_location_id' => '299 to show the fall back.'